Source/MediaStorage.swift

```diff
@@ -585,15 +592,28 @@ public class MediaStorage {
             )
         }
     }
-
-    private func replaceUserMeta (userMeta: String!) -> String{
+    
+    private func getAccessToken(auth: AuthClient) -> String? {
+        let semaphore: dispatch_semaphore_t = dispatch_semaphore_create(0)
+        
+        var accessToken = ""
```

Comment:
Nope.
This variable should not be Optional.
Because this method does not need nil decision.